exe/docker_startup.sh

```diff
@@ -26,7 +26,7 @@ function development {
 function production {
   cd /app && npm run build && mkdir -p public/assets/js && mv dist/static/js/* public/assets/js
 
-  bundle exec rake db:create || true
+  # bundle exec rake db:create || true
```

Comment:
I will delete it, we will need to make sure that production db can be created by docker compose in production environment. It should happen, as environment variable that triggers it is the same for kubernetes and docker compose